backend/app/Http/Controllers/Auth/EmailVerificationNotificationController.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
rename file to EmailVerificationController and move the function in the VerifyEmailController file to here. so that we will have one controller for email verification